tests/unit/test_sql_execution.py

```
@@ -940,3 +940,88 @@ def test_federated_auth_params_bigquery_missing_params(
+        sql_alchemy_dict = {
+            "url": "postgresql://user:password@localhost/db",
+            "params": {"mssql_trusted_connection": True},
+        }
+        original_url = sql_alchemy_dict["url"]
+
+        _handle_mssql_trusted_connection(sql_alchemy_dict)
+
+        mock_logger.warning.assert_called_once_with(
+            "mssql_trusted_connection is only supported for MS SQL Server connections"
+        )
+        self.assertEqual(sql_alchemy_dict["url"], original_url)
```

Ruff (0.14.11)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)